### DISCUSSION TITLE
How can I deploy this to Vercel as a standalone server api/microservice?

### DISCUSSION THREAD

**Discussion by steve-rodri** [sofn-xyz/mailing#186] [Q&A] [answered]
Just having trouble following the instructions. Would be helpful to have some deployment instructions.

**Comment by psugihara** [comment, marked as answer]
Hey @steve-rodri, thanks for trying mailing.

`mailing server build` builds a production next app in your `.mailing` directory

Assuming mailing.config.json was in your root directory, on vercel you'd configure Build & Development settings with

Framework preset: Next.js
Build command: `npx mailing server build`
Output dir: `.mailing/.next`

I added an issue to make this more automatic with a `vercel.json` in #190 


cc @alexfarrill
